### PR TITLE
DX-2086: Add circle-plus-filled.svg

### PR DESCRIPTION
**Pull request by cormickjbrowne**

### CODE REVIEW

File: src/circle-plus-filled.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg class="dibs-vg-svg dibsvg-circle-plus-filled" version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 250 250">
```
Comment (nameofname) [inline src/circle-plus-filled.svg]: this class name should be "dibsvg-svg"

Comment (nameofname) [inline src/circle-plus-filled.svg]: derp, didn't mean to approve..

File: src/circle-plus-filled.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+    <path d="M250,124.940715 C250,193.916268 193.961988,250 124.994613,250 C56.0487868,250 0,193.916268 0,125.048506 C0,56.1699651 56.0487868,0 124.897643,0 C193.843469,0 250,55.9543828 250,124.940715 Z M134.59359,75 L115.266812,75 L115.266812,113.660113 L75,113.660113 L75,132.984793 L115.266812,132.984793 L115.266812,174 L134.59359,174 L134.59359,132.984793 L174,132.984793 L174,113.660113 L134.59359,113.660113 L134.59359,75 Z" id="Combined-Shape"></path>
```
Comment (hellatan) [inline src/circle-plus-filled.svg]: remove the `id` attribute at the end

**Comment by cormickjbrowne** [comment]
Bumped to version 1.37.0

**Pull request merged**